### PR TITLE
docs: update the storage page

### PR DESCRIPTION
**Pull request by Barbapapazes**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/1.guide/4.storage.md [hunk 1]
```diff
@@ -6,24 +6,35 @@ icon: ri:database-2-line
...
-`useStorage()` is an instance of [createStorage](https://unstorage.unjs.io/usage) using the [memory driver](https://unstorage.unjs.io/drivers/memory).
+`useStorage()` is an instance of [createStorage](https://unstorage.unjs.io/usage) using the [memory driver](https://unstorage.unjs.io/drivers/memory) by default. It's an auto-imported utility, so you can use it anywhere in your handler.
 
-**Example:** Simple (in memory) operations
+By default, Nitro will also mount the current directory using the filesystem driver, using a prefix and where each folder serve to extend the prefix.
 
-```js
+- Every file will be under the `root` base.
+- Every source file will be under the `src` base. (By default, `src` is the `root` base)
+- Every build (inside .nitro) will be under the `build` base.
```
Comment (pi0) [inline docs/content/1.guide/4.storage.md:15]: It is only available for development only and I consider it kinda more advanced usage. We might move it to the last section in the page

File: docs/content/1.guide/4.storage.md [hunk 2]
```diff
@@ -110,46 +120,15 @@ export default defineNuxtConfig({
...
 ::
 
+::alert{type="warning"}
+This is a temporary workaround, with a better solution coming in the future! Keep a lookout on the GitHub issue [here](https://github.com/unjs/nitro/issues/1161#issuecomment-1511444675).
+::
 
-Usage:
-
-```js
-await useStorage('redis').setItem('foo', { hello: 'world' })
```
Comment (pi0) [inline docs/content/1.guide/4.storage.md:117]: Why removing examples?

Comment (Barbapapazes) [reply docs/content/1.guide/4.storage.md:117]: I refactor it and move it to the top (line 20).

Comment (Barbapapazes) [reply docs/content/1.guide/4.storage.md:117]: It was redondant with the beginning.

**Pull request merged**